test/uk/gov/hmrc/merchandiseinbaggage/pact/VerifyContractSpec.scala

```diff
@@ -29,7 +29,9 @@ import scala.concurrent.duration._
 
 class VerifyContractSpec extends PactVerifySuite with CoreTestData {
 
-  private val pactDir = scala.util.Properties.envOrElse("PACTTEST", "../merchandise-in-baggage-frontend/pact/")
+  private val pactDir                = scala.util.Properties.envOrElse("PACTTEST", "../merchandise-in-baggage-frontend/pact/")
+  private val currencyConversionStub = app.injector.instanceOf[CurrencyConversionStub]
```

Comment:
Please revert these changes as `EoriCheckStub` and `CurrencyConversionStub` don't need to be classes, thanks.